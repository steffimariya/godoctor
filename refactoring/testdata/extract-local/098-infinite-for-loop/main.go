package main

import "fmt"

func main() {
	a := 5
	b := 7
	for {
		c := a + b // <<<<< extractLocal,9,7,9,12,newVar,pass
		fmt.Println("c is", c)
		if c == 12 {
			break
		}
	}
}
