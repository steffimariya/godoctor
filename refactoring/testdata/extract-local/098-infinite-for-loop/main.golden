package main

import "fmt"

func main() {
	a := 5
	b := 7
	for {
		newVar := a + b
		c := newVar // <<<<< extractLocal,9,7,9,12,newVar,pass
		fmt.Println("c is", c)
		if c == 12 {
			break
		}
	}
}
